.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,41 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ "main" ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    # https://github.com/docker/setup-buildx-action
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+
+    # https://github.com/docker/login-action
+    - name: Login to Docker Hub
+      uses: docker/login-action@v3
+      with:
+        username: ${{ vars.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+    # https://github.com/marketplace/actions/docker-hub-description
+    - name: Update repo description
+      uses: peter-evans/dockerhub-description@v4.0.0
+      with:
+        username: ${{ vars.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+        repository: hackinsdn/secflood
+        short-description: ${{ github.event.repository.description }}
```

Comment:
Let's skip this for now, because the description is pretty static and we can update it on Docker Hub once for good.

Reply:
Nice catch :)
Just rebased to remove this commit for good.